extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -770,7 +770,7 @@ private EmbeddedInfo getEmbeddedInfo() throws ParsingException, ReCaptchaExcepti
 
         } catch (IOException e) {
             throw new ParsingException(
-                    "Could load deobfuscation code form restricted video for the Youtube service.", e);
+                    "Could not load deobfuscation code for YouTube restricted video", e);
```

Comment:
Heh, good catch, that "not". I'd remove the "restricted", though:

```suggestion
                    "Could not load deobfuscation code for YouTube video", e);
```

Reply:
Oh, yes, this is not only used for restricted videos anymore